modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a Route via an Ingress
+
+Some ecosystem components have an integration with Ingress resources but not with
+Route resources. To cover for this case, Openshift will automatically create
+managed Route object(s) when an Ingress is created. These Route objects are deleted
+upon deletion of the corresponding Ingress.
+
+.Procedure
+
+Define an Ingress object:
++
+.YAML Definition of an Ingress
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: frontend
+  annotations:
+    route.openshift.io/termination: "edge" <1>
+spec:
+  rules:
+  - host: www.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: frontend
+          servicePort: 443
+  tls:
+  - hosts:
+    - www.example.com
+    secretName: example-com-tls-certificate
+----
++
+
+<1> The `route.openshift.io/termination` annotation can be used to configure the `termination` of the `Route`
+as `Ingress` has no property for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All
+other values will be silently ignored. When unset, `edge` will be used.
+
+Apply it to your cluster:
++
+----
+$ oc apply -f ingress.yaml
+----
++
+
+List your routes:
++
+----
+$ oc get routes
+----
++
+
+The result will include an autogenerated route whose name starts with `frontend-`:
++
+----
+NAME             HOST/PORT         PATH    SERVICES    PORT    TERMINATION          WILDCARD
+frontend-gnztq   www.example.com           frontend    443     reencrypt/Redirect   None
```

Comment:
The yaml for the ingress specified `route.openshift.io/termination: "edge"`, so the termination here should be `edge/Redirect`, right?  Or maybe it would be better to change the example to specify `route.openshift.io/termination: "redirect"`.